VideoLocker/res/layout/player_controller.xml

```diff
@@ -1,224 +1,204 @@
-    </LinearLayout>
-
-    <!-- Put play/pause button to the center of player -->
-
-    <RelativeLayout
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:layout_above="@id/panel_bottom"
-        android:layout_below="@id/video_top_bar"
-        android:layoutDirection="ltr"
-        android:paddingLeft="10dp"
-        android:paddingRight="10dp" >
+            android:layout_gravity="center"
+            android:paddingTop="4dip"
+            android:text="@string/lbl_slash"
+            android:textSize="12sp" />
 
-        <ImageButton
-            android:id="@+id/pause"
+        <org.edx.mobile.view.custom.ETextView
+            android:id="@+id/time"
+            style="@style/semibold_white_text"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_centerInParent="true"
-            android:background="@drawable/ic_pause_button_selector"
-            android:contentDescription="@string/app_name" />
+            android:layout_gravity="center"
+            android:paddingEnd="4dip"
+            android:paddingLeft="0dp"
+            android:paddingRight="4dip"
+            android:paddingStart="0dp"
+            android:paddingTop="4dip"
+            android:textSize="12sp"
+            android:textStyle="bold"
+            tools:targetApi="17"
+            tools:text="02:05" />
+
+        <org.edx.mobile.third_party.iconify.IconButton
+            android:id="@+id/settings"
+            style="@style/video_player_bottom_control"
+            android:layout_width="@dimen/video_player_bottom_control_width"
+            android:layout_height="@dimen/video_player_bottom_control_height"
+            android:contentDescription="@string/video_player_settings"
+            app:iconColor="@color/edx_grayscale_neutral_white_t"
+            app:iconName="fa_cog" />
+
+
+        <org.edx.mobile.third_party.iconify.IconButton
+            android:id="@+id/fullscreen"
+            style="@style/video_player_bottom_control"
+            android:layout_width="@dimen/video_player_bottom_control_width"
+            android:layout_height="@dimen/video_player_bottom_control_height"
+            android:contentDescription="@string/video_player_enter_fullscreen"
+            app:iconColor="@color/edx_grayscale_neutral_white_t"
+            app:iconName="fa_compress"
+            tools:targetApi="17" />
+    </LinearLayout>
 
-    </RelativeLayout>
+    <!-- Put play/pause button to the center of player -->
 
-</RelativeLayout>
\ No newline at end of file
+    <org.edx.mobile.third_party.iconify.IconButton
+        android:id="@+id/pause"
+        android:layout_width="@dimen/video_player_pause_btn_width"
+        android:layout_height="@dimen/video_player_pause_btn_height"
+        android:layout_gravity="center"
+        android:background="@null"
+        android:contentDescription="@string/video_player_pause"

```

Comment:
lets leave this for now, as its the only implementation and play/pause style usage is far fetched elsewhere, unlike progress/seek bar.